src/docker_compose.rs

```diff
@@ -189,41 +194,82 @@ fn generate_docker_compose_content(dockerfile: PathBuf, dev_conf: &Option<Dev>)
                 ),
                 (
                     POSTGRES_SERVICE_NAME.into(),
-                    Some(Service {
-                        image: Some(service_images.postgres),
-                        environment: Some(Environment::KvPair(postgres_envs.into())),
-                        healthcheck: Some(Healthcheck {
-                            test: Some(HealthcheckTest::Multiple(vec![
-                                "CMD".into(),
-                                "pg_isready".into(),
-                                "-U".into(),
-                                "postgres".into(),
-                            ])),
-                            start_period: Some("10s".into()),
-                            interval: Some("5s".into()),
-                            timeout: None,
-                            retries: 5,
-                            disable: false,
-                        }),
-                        ..Default::default()
+                    Some(if expose {
```

Comment:
You should try to define a if at the ports field definition, or create a variable with the ports filed value instead of having a if arround the service.